Dockerfile

```diff
@@ -40,6 +40,14 @@ RUN wget ${BAZEL_DOWNLOAD_URL}/${BAZEL_VERSION}/${BAZEL_INSTALLER} && \
 RUN pip install --upgrade pip setuptools wheel && \
     pip install pipenv
 
+# Check for python version and install black
+
+RUN if [ ${PYTHON_VERSION} != 3.5* ]; then pip install black; fi
+
+# RUN v="$(python3 -c "print(__import__('sys').version_info[2])")" && \
```

Comment:
Is this commented part of the code needed? Can we remove this?

Reply:
No, it is not required. I was going to fix it once all the tests are passed.